### ISSUE TITLE
Error after fresh installation - Laravel 8

### ISSUE THREAD

**Issue by james-lukensow** [coolsam726/jetstream-inertia-generator#23]
**Describe the bug**
Performed a fresh installation with Laravel 8.

Loading the home page, I'm returned the following error: 

`nnocenzi\Vite\Exceptions\ManifestNotFound
The manifest could not be found. Did you start the development server? Tried: /var/www/html/public/build/manifest.json (View: /var/www/html/resources/views/app.blade.php)`

**Expected behavior**
Page loads correctly.

**Screenshots**
![Screen Shot](https://uc3b05f85f8f919607bedfa95737.previews.dropboxusercontent.com/p/thumb/ABRUO07kiYguS_kbgkWChlrW91xlOotp_0tPeMsRXP9Joxa3fB7iurKd1_DEDHT3cCsttwAkQRSoq3UwYJ0V-6IIEKhYD5BOgCRwZh07rx0pn_Z2yjPx3FGurZQANnu9ftX5Lmzq4L0GiIjQJbb7pOXnifpc40YwruVdq83vO29kgIurHejnosP3FPBhpw2isUHvrosLSyAqfHd_Obl1SY4DV93EM2WeSkdrOL2n3dYkVrIcSJ0VlxwdWc9KbG4aBstzZeQTLF2LxCDZoyRzACeWLtDLsoiax_5oMRnUyRA28ZPYUnvdXc7iPL_38pMuVhbmy8qop8Gap5tkm4tetTIISuXjtAjepvCA9M9QmIskJwCQ0y67wrwZvvZFEZa0HSAHhqmsBU2Cp2HYw8R31NgA/p.png?fv_content=true&size_mode=5)

**Comment by coolsam726** [comment]
Hi @james-lukensow , 
Did you run `yarn dev` or `yarn prod` after the installation?

**Comment by coolsam726** [comment]
I have added Step 11 of the installation procedure to capture that. If after that you still have issues, feel free to reach out.

**Comment by james-lukensow** [comment]
@coolsam726  I've tried running `npm run dev`, etc.

I think the issue, could lie with the fact that I'm using Laravel Sail, and the issue is on the docker side. After I posted the above issue, I remembered that I did the initial install via my mac, and after installing your package, I removed node_modules, and via sail, ran `sail npm install`. I'm getting some GYP issues, which leads me to believe the issue lies on the docker side.

Sample...

`npm ERR! code 1
npm ERR! git dep preparation failed
npm ERR! command /usr/bin/node /usr/lib/node_modules/npm/bin/npm-cli.js install --force --cache=/home/sail/.npm --prefer-offline=false --prefer-online=false --offline=false --no-progress --no-save --no-audit --include=dev --include=peer --include=optional --no-package-lock-only --no-dry-run
npm ERR! npm WARN using --force Recommended protections disabled.`

And the errors go on for awhile...
